content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+The queuing configuration allows tuning the fair queuing algorithm for a
+priority level. Details of the algorithm can be read in the
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md),
+but in short:
+
+* Increasing `queues` reduces the rate of collisions between different flows, at
+  the cost of increased memory usage. A value of 1 here effectively disables the
+  fair-queuing logic, but still allows requests to be queued.
+
+* Increasing `queueLengthLimit` allows larger bursts of traffic to be sustained
+  without dropping any requests, at the cost of increased memory usage.
+
+* Changing `handSize` allows you to adjust the probability of collisions between
+  different flows and the overall concurrency available to a single flow in an
+  overload situation, traded off against resource consumption. 
+    {{< note >}}
+    The probability that two flows with different flow distinguishers
+    can starve each other of resources is `1 / choose(queues, handSize)`. The choice
+    of `handSize` that minimizes this probability is therefore `Queues / 2`. However,
+    much smaller values of `handSize` will in practice make such collisions very
+    unlikely while consuming less resources. A larger `handSize` also increases the
+    effective concurrency share available to a single flow in an overload situation.
+    {{< /note >}}
+
+### FlowSchema
+A FlowSchema matches some inbound requests and assigns them to a
+PriorityLevelConfiguration. Every inbound request is matched against every
+FlowSchema in turn, starting with those with highest `MatchingPrecedence` and
+working down. 
+
+{{< caution >}}
+Only the first matching FlowSchema for a given request matters.
+If multiple FlowSchemas match a single inbound request, it will be assigned
+based on the one with the highest `MatchingPrecedence`.
+{{< /caution >}}
+
+A FlowSchema matches a given request if at least one of its `rules`
+matches. A rule matches if at least one of its `subjects` *and* at least
+one of its `resourceRules` or `nonResourceRules` (depending on whether the
+incoming request is for a resource or non-resource URL) matches. 
+
+For the `name` field in subjects, and the `verbs`, `apiGroups`, `resources`,
+`namespaces`, and `nonResourceURLs` fields of resource and non-resource rules,
+the wildcard `*` may be specified to match all of the appropriate object,
+effectively removing it from consideration.
+
+A FlowSchema's `flowDistinguisherMethod` determines how requests matching that
+schema will be separated into flows for the fair queuing algorithm. It may be
+either `ByUser`, in which case one requesting user will not be able to starve
+other users of capacity, or `ByNamespace`, in which case requests for resources
+in one namespace will not be able to starve requests for resources in other
+namespaces of capacity. The correct choice for a given FlowSchema depends on the
+resource and your particular environment.
```

Comment:
1. A FlowDistinguisherMethod may, in the future, be expanded to have more than just the Type field that it has today.  The text here is talking about the value in the Type field, so should be clear about that.

2. There is also the option to omit the flow distinguisher method, in which case the flow distinguisher string will always be the empty string.

3. Note also that the earlier identification of "flow" with "source" is right only when the flow distinguisher method type is `ByUser`.

Reply:
The documentation style guide prohibits forward-looking statements. I've corrected the field name though.

Added a note about the empty-string option.

I believe I've cleaned up most discussion of "sources" and the document should now consistently refer to "flows".